tests/xfstests/enable_kdump.pm

```diff
@@ -16,6 +16,7 @@ use 5.018;
 use warnings;
 use base "opensusebasetest";
 use utils qw(power_action zypper_call);
+use serial_terminal "select_virtio_console";
```

Comment:
please follow the format to use single quotes `'select_virtio_console'` in all your changed files